rpc-jobs/multi-node-aio.yml

```diff
@@ -1,48 +1,63 @@
-- job:
-    name: 'OnMetal_Multi_Node_AIO'
+- project:
+    name: 'Multi-Node-AIO-Jobs'
+    series:
+      - mitaka:
+          branch: mitaka-13.1
+          OPENSTACK_ANSIBLE_BRANCH: "stable/mitaka"
+      - newton:
+          branch: newton-14.0
+          OPENSTACK_ANSIBLE_BRANCH: "stable/newton"
+      - master:
+          branch: master
+          OPENSTACK_ANSIBLE_BRANCH: "stable/newton"
+    context:
+      - xenial:
+          DEFAULT_IMAGE: "16.04.2"
+      - trusty:
+          DEFAULT_IMAGE: "14.04.5"
+    exclude:
+      - series: mitaka
+        context: xenial
+    jobs:
+      - 'OnMetal_Multi_Node_AIO_{series}-{context}'
+
+- job-template:
+    name: 'OnMetal_Multi_Node_AIO_{series}-{context}'
     project-type: workflow
     build-discarder:
       days-to-keep: 30
     concurrent: true
+    triggers:
+      - timed: "H 3 * * 1-5"
```

Comment:
Do we want to run these all at 3 AM?  I know the minute will be spread out but we'll still have how many instances in this matrix all running at the end of 3 AM.  Just concerned there may be capacity issues if someone is trying to spin up on-commit tests also?